src/commands/coverage/__tests__/upload.test.ts

```diff
@@ -302,17 +307,6 @@ describe('execute', () => {
     expect(output[2]).toContain(`Will upload code coverage report file ${path}`)
   })
 
-  test('with git metadata without argument (default value is true)', async () => {
-    const {context, code} = await runCLI([
-      '--verbose',
-      process.cwd() + '/src/commands/coverage/__tests__/fixtures/single_file.xml',
-    ])
-    const output = context.stdout.toString().split(os.EOL)
-    expect(id).toHaveBeenCalled()
-    expect(code).toBe(0)
-    expect(output[4]).toContain('Syncing git metadata')
-  })
```

Comment:
did we change the default for git metadata upload?

Reply:
yes, changed the behaviour to skip git upload by default. Looking at your other comment, I think it makes sense to remove it altogether - it is true that we're not using it